src/runtime/virtcontainers/remote.go

```diff
@@ -32,18 +32,20 @@ type remoteHypervisor struct {
 type remoteHypervisorSandboxID string
 
 type remoteService struct {
-	conn   *grpc.ClientConn
-	client pb.HypervisorClient
+	conn   net.Conn
+	client pb.HypervisorService
```

Comment:
@stevenhorsman HypervisorClient (struct) implements HypervisorService (Interface), so it compiles. Any special reason to replace the type from the struct to interface?

Reply:
hmmm there is a hypervisorClient (lower case 'h') in https://github.com/kata-containers/kata-containers/pull/8520/commits/613c75ba8ce1deb78913ac0929776b2bdf98ac33#diff-0df604aba3c29a070a7ed40486ce8193ecb2a36c2dabf91bb29e7627cff969d8R52 . This should be a private date structure so never mind.